Week-12/todo-app/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import TodoList from "./Components/TodoList";
+import TodoForm from "./Components/TodoForm";
+import "./App.css";
+
+const App = () => {
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (text) => {
+    const newTodo = {
+      id: Date.now(),
+      text: text,
+    };
+    setTodos([...todos, newTodo]);
+  };
+
+  const deleteTodo = (id) => {
+    const updatedTodos = todos.filter((todo) => todo.id !== id);
```

Comment:
<img width="691" alt="image" src="https://github.com/pesto-students/p10-harsh-srivastava-abhinavsingh06/assets/127043221/98979a54-0fff-4078-a061-7b98725d2ac0">
completed todos should move down, also its hard to figure out which are completed based on styling